misc/theme-verifier/pom.xml

```diff
@@ -72,6 +72,17 @@
             <version>2.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
+            <artifactId>owasp-java-html-sanitizer</artifactId>
+            <version>20240325.1</version>
```

Comment:
This `pom.xml` file is not consistent on how versions are defined. Not a blocker, but I guess we want to keep consistency with other pom files where versions are defined in the `properties` section.

Reply:
Yes, I'll rework that in a follow-up PR. I also received feedback from @ASzc that I should use the parent pom. Once we do this, we'll also use the centralized dependency management.